### PR TITLE
Move fcrepo-connector-file to its own repository.

### PR DESCRIPTION
**Pull request by whikloj**
Partially resolves [FCREPO-2028](https://jira.duraspace.org/browse/FCREPO-2028)

### CODE REVIEW

**Comment by acoburn** [comment]
👍

**Comment by ajs6f** [comment]
Are there going to be other PRs that accompanying this 'un?

**Comment by acoburn** [comment]
@ajs6f this repo will move to fcrepo4-exts: https://github.com/whikloj/fcrepo-connector-file
and there will be a PR on fcrepo4-webapp-plus

**Comment by ajs6f** [comment]
https://www.youtube.com/watch?v=XwozVKOkzz4

**Comment by whikloj** [comment]
Related PR https://github.com/fcrepo4-exts/fcrepo-webapp-plus/pull/49

File: fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java [hunk 1]
```diff
@@ -1774,28 +1773,6 @@ public void testResponseContentTypes() throws Exception {
...
 
-    /**
-     * Given a directory at: test-FileSystem1/ /ds1 /ds2 /TestSubdir/ and a projection of test-objects as
-     * fedora:/files, then I should be able to retrieve an object from fedora:/files/FileSystem1 that lists a child
-     * object at fedora:/files/FileSystem1/TestSubdir and lists datastreams ds and ds2
-     *
-     * @throws IOException thrown during this function
-     */
-    @Test
-    public void testGetProjectedNode() throws IOException {
```
Comment (awoods) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: Ideally, we would not be losing the tests in `FedoraLdpIT.java` and `FedoraNodesIT.java`, but they would instead be found in the `fcrepo-connector-file` or `fcrepo-webapp-plus` projects.
Hopefully the future maintainers of this effort will bring these tests back.

Comment (acoburn) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: Oh, but @awoods we're not losing any of those tests. That's what https://github.com/fcrepo4-exts/fcrepo-connector-file/pull/1 was all about. All the iTests that were in the -http-api module are now in this project.

Sometimes you get to have your cake and eat it, too.

Comment (awoods) [inline fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java]: I lost those test methods in: https://github.com/fcrepo4-exts/fcrepo-connector-file/blob/master/src/test/java/org/fcrepo/integration/connector/file/FileConnectorIT.java#L128

That's great.

**Comment by acoburn** [comment]
Yep, they are all there. It took some doing to get them to run properly, but @whikloj and I felt that it was more appropriate to have them in fcrepo-connector-file than in fcrepo-webapp-plus

**Pull request merged**